tests/integration/utils/test_mssql_log_utils.py

```diff
@@ -58,7 +58,7 @@ def log(event, **kwargs):
 
     assert results == [(1,), (3,)]
 
-    assert log_lines[0] == ("[info   ] SQL:\n" "          SELECT 1")
+    assert log_lines[0] == ("[info   ] SQL:\n          SELECT 1")
```

Comment:
Could lose the brackets now I think. It's just a bit confusing to have them when they're not doing anything.